packages/frontend/src/config/environmentDefaults/development.js

```diff
@@ -50,5 +50,6 @@ export default {
     FARMING_CLAIM_YOCTO: '1',
     REF_FINANCE_API_ENDPOINT: 'https://dev-indexer.ref-finance.com/',
     REF_FINANCE_CONTRACT: 'ref-finance-101.testnet',
-    USN_CONTRACT: 'usdn.testnet'
+    USN_CONTRACT: 'usdn.testnet',
+    HAPI_PROTOCOL_ADDRESS: 'proxy.contracts.sergei24.testnet'
```

Comment:
can you elaborate on what proxy.contracts.sergei24.testnet is?

Reply:
ah i see it's the constant provided by the [near proxy contract](https://github.com/HAPIprotocol/near-proxy-contract)